apps/web/components/AboutLatestNews/AboutLatestNews.tsx

```diff
@@ -125,9 +126,18 @@ const BigNewsItem = ({
         {news.title}
       </Text>
       <Text variant="intro">{news.intro}</Text>
-      <ArrowLink as={as} href={href}>
-        {readMore}
-      </ArrowLink>
+      <Link as={as} href={href}>
+        <Button
+          colorScheme="default"
+          icon="arrowForward"
+          iconType="filled"
+          size="default"
+          type="button"
+          variant="text"
+        >
+          {readMore}
+        </Button>
```

Comment:
there is no need to specify default values
```suggestion
        <Button
          icon="arrowForward"
          variant="text"
        >
          {readMore}
        </Button>
```